installer/install.iss

```diff
@@ -2292,6 +2292,7 @@ begin
     if DetectNetFxVersion()<378675 then begin
         RdbGitCredentialManager[GCM].Checked:=False;
         RdbGitCredentialManager[GCM].Enabled:=False;
+        RdbGitCredentialManager[GCM].Caption:=RdbGitCredentialManager[GCM].Caption+' (requires .NET >= 4.5.1)'
```

Comment:
This should say `.NET Framework`. It can't really be confused for any other .NET product with these twi version numbers in Question, but it's better to use accurate names with how many different .NET products/standards exist nowadays.

Reply:
Thanks, addressed in https://github.com/git-for-windows/build-extra/pull/329/commits/5db650c0aec90ecd7e3c193bec51a18ad5c5ed90